packages/cli/src/databases/migrations/mysqldb/1646992772331-CreateUserManagement.ts

```diff
@@ -156,7 +156,7 @@ export class CreateUserManagement1646992772331 implements ReversibleMigration {
 		);
 
 		await queryRunner.query(
-			`INSERT INTO ${tablePrefix}settings (\`key\`, value, loadOnStartup) VALUES ("ui.banners.v1.dismissed", "true", 1)`,
+			`INSERT INTO ${tablePrefix}settings (\`key\`, value, loadOnStartup) VALUES ("ui.banners.dismissed", "[\"V1\"]", 1)`,
```

Comment:
this query is invalid. maybe we should start running DB tests automatically when ever the migrations folder is modified .

Reply:
If we can do it, I would be 100% for it. It's always hard for me to find a mysql testing env (sorry, completely forgot to mention that query needs testing 🤦‍♂️).